asf_heat_pump_suitability/getters/base_getters.py

```diff
@@ -59,6 +59,21 @@ def get_df_from_zip_csv_s3(path: str, extract_file: str, **kwargs) -> pl.DataFra
     return df
 
 
+def get_df_from_excel_s3_path(path: str, **kwargs) -> pl.DataFrame:
+    """
+    Get dataframe from Excel file stored in s3 path.
+
+    Args
+        path (str): S3 URI to Excel file
+        **kwargs for pl.read_excel()
+    Returns
+        pl.DataFrame: dataframe from Excel file
+    """
+    content = BytesIO(get_content_from_s3_path(path))
+    df = pl.read_excel(content, **kwargs)
```

Comment:
I needed to pip install `xlsx2csv`. So add this to the requirements

Reply:
oh I see you did this in the 48_ PR. Ignore here!

Reply:
Thanks for saying 👍 , I have a tendency to not commit the necessary changes for the packages in the requirements.txt!